### PR TITLE
Test cluster scoped

### PR DESCRIPTION
**Pull request by briangallagher**
# Issue link

https://issues.redhat.com/browse/MGDAPI-3308

### CODE REVIEW

File: controllers/tenant/rhoamtenant_controller.go [hunk 1]
```diff
@@ -27,6 +27,7 @@ var log = l.NewLoggerWithContext(l.Fields{l.ControllerLogContext: "tenant_contro
 // +kubebuilder:rbac:groups=integreatly.org,resources=rhoamtenant,verbs=get;list;watch
 // +kubebuilder:rbac:groups=integreatly.org,resources=rhoamtenant/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=user.openshift.io,resources=users,verbs=watch;get;list;update
+// +kubebuilder:rbac:groups="",resources=namespaces,verbs=get;list;watch
```
Comment (briangallagher) [inline controllers/tenant/rhoamtenant_controller.go:30]: not needed

**Comment by carlkyrillos** [comment]
Reviewed changes to files and confirmed that cluster scoped, multitenant installation of RHOAM works properly. Addressing the failing prow/multitenant-rhoam-e2e test will be tracked in a followup JIRA.
/lgtm

**Pull request merged**